doc/proxy-metrics.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Proxy Metrics"
+docpage = true
+[menu.docs]
+  parent = "proxy-metrics"
++++
+
+The Conduit proxy exposes metrics that describe the traffic flowing through the
+proxy.  These metrics are exposed in Prometheus format on the proxy control port
+(4190 by default) at the `/metrics` path.  The following metrics will be
+exposed:
+
+### `request_total`
+
+A counter of the number of requests the proxy has received.
+
+### `request_duration_ms`
+
+A histogram of the duration of a request.  This is measured from when the
+request headers are received to when the request stream has completed.
+
+### `response_total`
+
+A counter of the number of responses the proxy has received.
+
+### `response_duration_ms`
+
+A histogram of the duration of a response.  This is measured from when the
+response headers are received to when the response stream has completed.
+
+### `response_latency_ms`
+
+A histogram of the total latency of a response.  This is measured from when the
+request headers are received to when the response stream has completed.
```

Comment:
Do we also want to have handle time metric(s) as discussed https://github.com/runconduit/conduit/issues/479#issuecomment-369386203?

Reply:
To keep a handle on scope I don't think I want to add it to the docs now.  However, it's definitely a valuable metric and when we add the metric we should add it to this doc as well.

Some questions that spring to mind about handle time:
* What units would handle time be reported in?  ms should almost always be zero, but mixing units (ms for latency, us for handle time) is confusing.
* Do the latency and duration metrics include the handle time?
* If we add handle time we probably also want to add total time (total time =handle time + latency) or vice versa (service latency = latency - handle time) to discourage users from doing the wrong thing of trying to add or subtract percentiles.  Because math.